CorsixTH/Lua/hospital.lua

```diff
@@ -819,15 +803,7 @@ function Hospital:boilerBreakdown(broken_heat)
   heat_vars.heating_broke = true
 
   -- Only show the message when relevant to the local player's hospital.
-  if self:isPlayerHospital() then
-    if heat_vars.radiator_heat == 0 then
-      self.world.ui.adviser:say(_A.boiler_issue.minimum_heat)
-      self:coldWarning()
-    else
-      self.world.ui.adviser:say(_A.boiler_issue.maximum_heat)
-      self:hotWarning()
-    end
-  end
+  if self:isPlayerHospital() then self:warnBoilerBreakdown(broken_heat) end
```

Comment:
Don't call functions that don't exist in a class, ever.

It's fine to create empty a place-holder function in this class, and then re-implement it in the player hospital. That also allows you to remove the condition. Conceptually always warn, just some versions of "warn" don't do anything useful.